TSPL.docc/ReferenceManual/Types.md

```diff
@@ -948,6 +949,45 @@ https://github.com/apple/swift-evolution/blob/main/proposals/0335-existential-an
 >
 > *boxed-protocol-type* → **`any`** *type*
 
+## Type-Parameter Pack
+
+XXX OUTLINE:
+
+- `each T` creates a type-parameter pack
+  when it appears in a generic parameter clause,
+  or indicates which pack should be expanded
+  when it appears in a `repeat` expression
+
+- `repeat T` expands the type-parameter pack
+
+- Packs are never nested; expansion implies flattening
+
+- To expand the values, you use a parameter pack expression;
+  see <doc:Expressions#Parameter-Pack-Expression>
+
+- It's valid for a type pack to contain no elements.
+
+- list of places where type-parameter pack can appear:
+
+  + generic type parameter list
+  + as an argument to a generic type parameter
```

Comment:
Does it warrant including `tuple element` in this list? Or is that considered to be effectively captured by `generic type parameter list`?

Reply:
Tuple elements, and function type parameter lists, too.